pkg/secrets/provider_status.go

```diff
@@ -98,6 +103,13 @@ func (f fileUpdater) copyScripts() error {
 }
 
 func (f fileUpdater) copyFile(srcPath, destPath string) error {
+
+	// Create the directory
+	err := f.os.mkdirAll(f.scriptDestDir, os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("unable to mkdir at %q: %s", destPath, err)
+	}
```

Comment:
Why is this in `copyFile()` instead of `copyScripts()`? It seems like that's where we do most of the setup like constructing the destination URI. Wouldn't it make sense to create the directory there too?

Reply:
Now that you mention it... yes, I'll move it. Also it only needs to be created once, should more scripts be added.